app/services/streaming/streaming.ts

```diff
@@ -135,7 +135,7 @@ export class StreamingService extends StatefulService<IStreamingServiceState>
 
   private finishStartStreaming() {
     const shouldConfirm = this.settingsService.state.General.WarnBeforeStartingStream;
-    const confirmText = 'Are you sure you want to start streaming?';
+    const confirmText = $t('Are you sure you want to start streaming?');
```

Comment:
It seems like we didn't update the EN language file on this one.

Reply:
that's true for like 70% of strings in our repo lol i think that should be a separate task